app/assets/javascripts/controllers/current_deploys_ctrl.js

```diff
@@ -0,0 +1,52 @@
+samson.controller("currentDeploysCtrl", function($scope, $http, SseFactory, $interval, DeployHelper) {
+  'use strict';
+
+  $scope.deploys = [];
+  $scope.helper = DeployHelper;
+
+  function updateUpdatedAt() {
+    _.each($scope.deploys, function(deploy) {
+      deploy.updated_at_ago = moment(deploy.updated_at).fromNow();
+    });
+  }
+
+  function init() {
+    $http.get('/deploys/active.json').success(function(result) {
+      $scope.deploys = result.deploys;
+      updateUpdatedAt();
+    });
+  }
+
+  $scope.addDeploy = function(deploy) {
+    $scope.deploys = [deploy].concat($scope.deploys);
```

Comment:
Yup its an array so the angular directive displays it in the same order.
On 6 May 2015 1:27 am, "Roman Sandler" <notifications@github.com> wrote:

> In app/assets/javascripts/controllers/current_deploys_ctrl.js
> <https://github.com/zendesk/samson/pull/394#discussion_r29727091>:
>
> > +
> > +  function updateUpdatedAt() {
> > +    _.each($scope.deploys, function(deploy) {
> > +      deploy.updated_at_ago = moment(deploy.updated_at).fromNow();
> > +    });
> > +  }
> > +
> > +  function init() {
> > +    $http.get('/deploys/active.json').success(function(result) {
> > +      $scope.deploys = result.deploys;
> > +      updateUpdatedAt();
> > +    });
> > +  }
> > +
> > +  $scope.addDeploy = function(deploy) {
> > +    $scope.deploys = [deploy].concat($scope.deploys);
>
> does prepending rather than appending affect the displayed order?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/394/files#r29727091>.
>